### ISSUE TITLE
[macOS] Garbled graphics when spamming two light effects at once.

### ISSUE THREAD

**Issue by WhosPix3l** [mat1jaczyyy/apollo-studio#407]
### Description
When spamming two buttons that has light effects, Apollo will freeze, and garble visuals.

### To Reproduce
Create a new light effects that fills the whole Launchpad.

### Expected behavior
Apollo doesn't freeze.

### Your setup
Apollo Studio 1.8.2, macOS Catalina

### Additional context
Fast LED mod for MK2, 100 FPS limit on Apollo
![VID-64460529-160613-079](https://user-images.githubusercontent.com/66474423/95802557-02b1ed00-0d28-11eb-99c8-ff876b60c9eb.gif)

**Comment by mat1jaczyyy** [comment]
attach a pf that reproduces this...
"to reproduce" => i need to be able to read this and easily get the exact same thing as you
"Create a new light effects that fills the whole Launchpad." there are so many factors that i have to guess that might affect this

**Comment by mat1jaczyyy** [comment]
honestly looks like a cable connection bug. mk2 fast led mod windows 1.8.2 cant repro.
@Brendonovich can you repro?

**Comment by mat1jaczyyy** [comment]
@WhosPix3l can you still repro this in 1.8,3?

**Comment by WhosPix3l** [comment]
> @WhosPix3l can you still repro this in 1.8,3?

will try soon on hackintosh once i get home

**Comment by WhosPix3l** [comment]
yea still happening

**Comment by WhosPix3l** [comment]
this might also relate to this issue but might just be my hardware: apollo lights are sluggish if an electron app / chromium browser is open

**Comment by mat1jaczyyy** [comment]
is your pc overstressed? what are your specs? there's no way we can guarantee correct behavior if your pc is maxed out...

**Comment by WhosPix3l** [comment]
yeah, actually i think my computer is just bad lol, the garbled effects are still there however

**Comment by WhosPix3l** [comment]
seems to be better now, a workaround is to close apollo's editor, but it seems like the lights are still faster under windows than macos
